docs/getting-started.md

```diff
@@ -123,7 +123,7 @@ tar.exe xvf .\containerd-$Version-windows-amd64.tar.gz
 Copy-Item -Path .\bin -Destination $Env:ProgramFiles\containerd -Recurse -Force
 
 # add the binaries (containerd.exe, ctr.exe) in $env:Path
-$Path = [Environment]::GetEnvironmentVariable("PATH", "Machine") + [IO.Path]::PathSeparator + "$Env:ProgramFiles\containerd"
+$Path = [Environment]::GetEnvironmentVariable("PATH", "Machine") + [IO.Path]::PathSeparator + "$Env:ProgramFiles\containerd\bin"
```

Comment:
I am not sure why we need this. The default install directory for contained on windows is C:\ProgramFiles\containerd. We do not need "\bin"

Reply:
Thank you for your review.
I installed containerd on Windows following the command.Then I saw the binaries (containerd.exe, ctr.exe) in the path($Env:ProgramFiles\containerd\bin).

So I think we could fix it in any way below:
- 1 
`$Path = [Environment]::GetEnvironmentVariable("PATH", "Machine") + [IO.Path]::PathSeparator + "$Env:ProgramFiles\containerd\bin`
- 2
`Copy-Item -Path .\bin\* -Destination $Env:ProgramFiles\containerd -Recurse -Force`